### PR TITLE
[Don't merge until DP Launch] Congrats: swap Dance Party for Minecraft

### PR DESCRIPTION
**Pull request by Erin007**
The last of this year's /congrats changes outline in the [spec](https://docs.google.com/document/d/1T_NY5ufkcPcmCnHSIoBtHF32SKmm5_t_nmxj0LrwPEs/edit#). 

Students younger than 13 who finish tutorials other than Dance, one of the Minecrafts or AppLab, will see a card promoting Dance Party rather than Minecraft Hero. 

BEFORE: 
screen shot 2018-11-01 at 11 59 43 am

AFTER: 
screen shot 2018-11-01 at 11 35 59 am

### CODE REVIEW

File: apps/i18n/common/en_us.json [hunk 1]
```diff
@@ -813,9 +815,6 @@
   "minecraftAquaticButton": "Go to Minecraft Education",
   "minecraftAquaticDesc": "Continue learning with Minecraft.",
   "minecraftAquaticTitle": "Minecraft Education",
-  "minecraftMarketingButton": "Start",
-  "minecraftMarketingDesc": "Minecraft is back for the Hour of Code with a brand new activity! Journey through Minecraft with code.",
-  "minecraftMarketingTitle": "Minecraft: Hero’s Journey",
```
Comment (maddiedierker) [inline apps/i18n/common/en_us.json:818]: i'm sure you checked, but just double-checking that these "minecraft_" strings aren't used elsewhere

Comment (Erin007) [reply apps/i18n/common/en_us.json:818]: Yep, thanks for the reminder!

**Comment by Erin007** [comment]
Heads up @breville and @madelynkasula I just updated this PR to include the final copy (hooray!) for the Dance Party and Dance Party follow-up cards. 
screen shot 2018-11-02 at 2 17 46 pm
screen shot 2018-11-02 at 2 25 43 pm

**Comment by Erin007** [comment]
screen shot 2018-11-07 at 10 47 02 am
Updated the asset for Dance Party follow up!

**Comment by Erin007** [comment]
One last tweak to address issue: https://github.com/code-dot-org/dance-party/issues/295

**Pull request merged**